src/main/java/seedu/mindmymoney/helper/GeneralFunctions.java

```diff
@@ -35,8 +36,9 @@
      */
     public static String parseInputWithCommandFlag(String input, String startingFlag, String endingFlag)
         throws MindMyMoneyException {
-        if (!(input.contains(startingFlag + " ") && input.contains(" " + endingFlag))) {
-            throw new MindMyMoneyException("You are missing a flag or lack the spacing between the flags!");
+        if (!(input.contains(startingFlag + " ") && input.contains(endingFlag + " "))) {
```

Comment:
why swap the ending flag and the " "? my idea was that ending flag is going to appear like " /c"

Reply:
Sorry my bad, i editted the code previously and forgot to change it back correctly. Will make the necessary changes